cosmwasm/deployment/terra2/README.md

```diff
@@ -131,6 +133,11 @@ authority, so these don't have to go through governance.
 
 For example, to migrate the token bridge to 59614, run in `tools/`:
 
+<!-- cspell:disable -->
 ```sh
```

Comment:
don't think these three lines were intended to be duplicated

Reply:
I didn't change any existing stuff but have fixed it in dbc800d3c5d24370c3a2c3099f26c689b24f8271